merlin/dag/node.py

```diff
@@ -452,21 +452,22 @@ def export(
         version: int = 1,
     ):
         """
-        Export a Triton config directory for this node.
+        Export a directory for this node, containing the required artifacts
+        to run in the target context.
 
         Parameters
         ----------
         output_path : Union[str, os.PathLike]
-            The base path to write this node's config directory.
+            The base path to write this node's export directory.
         node_id : int, optional
             The id of this node in a larger graph (for disambiguation), by default None.
         version : int, optional
-            The Triton model version to use for this config export, by default 1.
+            The version of the node to use for this export, by default 1.
 
         Returns
         -------
         ModelConfig
```

Comment:
I think `ModelConfig` is Triton-specific too

Reply:
So what should we put here? We dont know what it would be in any other case...

Reply:
Or just remove the `Returns` section

Reply:
OK I removed the return all together as advised.